code/modules/chemistry/tools/beakers.dm

```diff
@@ -181,6 +181,10 @@
 	icon_state = "largebottle-burn"
 	initial_reagents = "silver_sulfadiazine"
 
+/obj/item/reagent_containers/glass/beaker/large/cyborg
+	shatter_chemically(var/projectiles = FALSE)
+		return FALSE
```

Comment:
this works, but I added a `shatter_immune` var to `/obj/item/reagent_containers/glass` to fix this for the chemicompiler ones

Reply:
makes sense, fixed!